components/env/utils.js

```diff
@@ -36,10 +36,10 @@ export function parseString(name, fallback, options) {
  */
 function getEnv(name, options = {}) {
   const { runtime } = { runtime: false, ...options };
-  name = `FRED_${name}`;
+  const envName = `FRED_${name}`;
```

Comment:
Nit: Could reduce confusion in line 42, where `envName` is next to `getEnv(name)`.


```suggestion
  const fullName = `FRED_${name}`;
```

Alternatively, `name` could be `suffix`.

```suggestion
  const envName = `FRED_${suffix}`;
```